distributed/tests/test_actor.py

```diff
@@ -618,6 +628,42 @@ def worker_tasks_running(dask_worker):
 
 
 def test_one_thread_deadlock():
+    class UsesCounter:
+        # An actor whose method argument is another actor
+
+        def do_inc(self, ac):
+            return ac.increment().result()
+
+    with cluster(nworkers=2) as (cl, w):
+        client = Client(cl["address"])
+        ac = client.submit(Counter, actor=True).result()
+        ac2 = client.submit(UsesCounter, actor=True, workers=[ac._address]).result()
```

Comment:
Why not use one worker in the cluster startup instead of setting worker restrictions?

Reply:
done